src/org/rascalmpl/unicode/UnicodeConverter.java

```diff
@@ -0,0 +1,12 @@
+package org.rascalmpl.unicode;
+
+public class UnicodeConverter {
+    public static String unicodeArrayToString(int[] chars) {
```

Comment:
There is a specific [`String() constructor`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#%3Cinit%3E(int%5B%5D,int,int)) that takes care of this.

Reply:
Thanks, removed this class after replacing calls to this method with the String constructor.